libminifi/test/unit/ProvenanceTestHelper.h

```diff
@@ -20,6 +20,62 @@
 
 #include "Provenance.h"
 #include "FlowController.h"
+#include "FlowFileRepository.h"
+
+/**
+ * Test repository
+ */
+class FlowTestRepository : public FlowFileRepository
```

Comment:
the FlowFileRepo construct to populate the default repo type and parameters for the repo 
/*!
	 * Create a new provenance repository
	 */
	FlowFileRepository()
	 : Repository(Repository::FLOWFILE, FLOWFILE_REPOSITORY_DIRECTORY,
			MAX_FLOWFILE_REPOSITORY_ENTRY_LIFE_TIME, MAX_FLOWFILE_REPOSITORY_STORAGE_SIZE, FLOWFILE_REPOSITORY_PURGE_PERIOD)
	{
	}